assets/component-mega-menu.css

```diff
@@ -49,13 +42,19 @@
 .mega-menu__link {
   color: rgba(var(--color-foreground), 0.75);
   display: block;
+  font-size: 1.2rem;
```

Comment:
I find this a little small 😬 
From what i'm seeing on other websites, having a bold or different color level 2 seems like the most common use case ([H&M](https://www2.hm.com/en_us/index.html), [Quicksilver](https://www.quiksilver-shop.ca/), [Fao Schwarz](https://faoschwarz.com/), etc). 
T

Reply:
Something is off here, from the original logic, they would be styled as L2s, not L3s when in presence of L2s and L3s

Reply:
I also find `1.2rem` way too small. I've looked at other websites ([H&M](https://www2.hm.com/en_us/index.html), [Quicksilver](https://www.quiksilver-shop.ca/), [Fao Schwarz](https://faoschwarz.com/), [Gap](https://www.gapcanada.ca/), [Simons](https://www.simons.ca/en), [Nordstrom](https://www.nordstrom.ca/)) and they're all using minimum of `13px`. This led me to test with `1.3rem` and it looks so much more readable. `1.3rem` would match our link sizes for product links, which I personally find at the limit of being easily readable. 

## 1.2rem
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160690353-bc8f7ee0-2d49-48d9-addb-dbeb49c69880.png">

## 1.3rem
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160690327-19bd7df0-d2d0-4d74-afe0-b7aa2aac8a9d.png">

It does lose some of the hierarchy, so I'm not sure if there's something we can do about it. 

I don't have the strongest opinion here, but just sharing my personal point of view as a user. I find it kinda hard to read menus in Dawn and Studios on desktop when using `1.2rem` fonts, and I'd prefer having less distinction between the hierarchy levels than barely being able to read without having to zoom / get closer to the screen

------

<details>
<summary>Other themes @ 12px</summary>
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160692375-f4090c7a-7b2e-483b-8474-4bd83e432185.png">
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160692540-974f2e98-2822-4f08-9a91-838959d8616c.png">
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160692684-696440ce-e442-4998-9ee4-d3290bc6b679.png">
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160692827-dba2558a-63d9-40aa-b0e7-c76d27ada0fe.png">
<img width="789" alt="image" src="https://user-images.githubusercontent.com/37168033/160692936-528ba61a-5138-48f6-b72f-66a037a6885f.png">
</details>